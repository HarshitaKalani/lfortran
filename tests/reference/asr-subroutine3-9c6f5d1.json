{
    "basename": "asr-subroutine3-9c6f5d1",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/subroutine3.f90",
    "infile_hash": "047260bb19e7393fa3d89883d9b5c6c0d9de0048f796f39b83dcc644",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-subroutine3-9c6f5d1.stderr",
    "stderr_hash": "813aa9575e71212e7cab0642ea42de822d5fc1459160ecba0343d64f",
    "returncode": 2
}