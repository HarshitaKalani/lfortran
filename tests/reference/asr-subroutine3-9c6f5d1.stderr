semantic error: Argument of `size` must be an array
 --> tests/errors/subroutine3.f90:8:18
  |
8 |         print *, size(f)
  |                  ^^^^^^^ 
